### PR TITLE
Recover explorer and fix module's bugs

### PR DESCRIPTION
**Pull request by tanbowensg**
Fix some bugs that will cause module unstable.

### CODE REVIEW

File: packages/runtime/src/components/_internal/ImplWrapper.tsx [hunk 1]
```diff
@@ -20,18 +20,24 @@ const _ImplWrapper = React.forwardRef<HTMLDivElement, ImplWrapperProps>((props,
...
+    if (!isInModule) {
+      eleMap.set(c.id, ele);
+    }
     hooks?.didDomUpdate && hooks?.didDomUpdate();
   };
 
   useEffect(() => {
-    if (eleRef.current) {
+    // If a component is in module, it should not have mask, so we needn't set it
+    if (eleRef.current && !isInModule) {
```
Comment (Yuyz0112) [inline packages/runtime/src/components/_internal/ImplWrapper.tsx:32]: is it possible to infer whether a component is in a module from the application schema?

Comment (tanbowensg) [reply packages/runtime/src/components/_internal/ImplWrapper.tsx:32]: No, because the component schema in module and application are the same.

**Comment by Yuyz0112** [comment]
LGTM, please resolve the conflicts, and then we can merge.

**Comment by tanbowensg** [comment]
Conlict Resolved.

**Review by Yuyz0112 — APPROVED** [review]

**Pull request merged**